### ISSUE TITLE
Adjust circle scaling for optimal viewing

### ISSUE THREAD

**Issue by aappling-usgs** [USGS-VIZLAB/water-use-15#196]
circles are probably too big right now

**Comment by jread-usgs** [comment]
I'm wondering what we have for ideas here. Getting too much smaller is going to leave us with pretty limited pattern on the national map, especially in the smaller categories. I bet we can go to ~65% current before this is an issue, but that doesn't solve our issue of thermoelectric on the northeast coast (#197 ) because they still aren't small enough there.

I am confident we can make things better than they are now, but I don't think we should have #197 as acceptance criteria for this one.

**Comment by aappling-usgs** [comment]
Agreed; as noted in #197, resolving this issue should only improve, not solve, the problems noted in #197.

**Comment by lindsaycarr** [comment]
OK, what are the general thoughts on this size change.

# USA, thermo, desktop

### Largest circle size = 15 (current)
![image](https://user-images.githubusercontent.com/13220910/39262576-60ffedcc-4885-11e8-9021-46326698449a.png)

### Largest circle size = 10
![image](https://user-images.githubusercontent.com/13220910/39262303-a9c1520e-4884-11e8-824b-7165375da2cb.png)

# USA, thermo, mobile

### Largest circle size = 15 (current)
![image](https://user-images.githubusercontent.com/13220910/39262647-9305db88-4885-11e8-8a64-ea74cdc70fd6.png)

### Largest circle size = 10
![image](https://user-images.githubusercontent.com/13220910/39262358-c6083a7c-4884-11e8-90d5-09bb6b69708a.png)

# New Jersey, thermo, mobile

### Largest circle size = 15 (current)
![image](https://user-images.githubusercontent.com/13220910/39262632-88439028-4885-11e8-9d1c-8cdf1b95c9e6.png)

### Largest circle size = 10
![image](https://user-images.githubusercontent.com/13220910/39262412-ee41b7b6-4884-11e8-97cc-91fb6d3bf4c8.png)

# USA, irrigation, desktop

### Largest circle size = 15 (current)
![image](https://user-images.githubusercontent.com/13220910/39262599-718b5b2c-4885-11e8-97b2-083eb947e2f8.png)

### Largest circle size = 10
![image](https://user-images.githubusercontent.com/13220910/39262476-178e2f32-4885-11e8-820a-5288bb687221.png)

**Comment by aappling-usgs** [comment]
I think it's an improvement, especially at the national view level.

For state views, what would you think about rescaling the circles to a different maximum? This could either be state specific (harder) or simply involve having a smaller maximum, e.g., largest circle size = 5

**Comment by lindsaycarr** [comment]
Hmm I guess that we are no longer bound to keeping them the same on zoomed scales...huh I think that might be the way to go!
